patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -2076,16 +2096,34 @@
  		for (int k = 0; k < num16; k++) {
  			KillTile_MakeTileDust(i, j, tile);
  		}
-@@ -45758,6 +_,9 @@
+@@ -45756,9 +_,16 @@
  
- 	private static void KillTile_DropItems(int x, int y, Tile tileCache, bool includeLargeObjectDrops = false)
+ 	private static Player GetPlayerForTile(int x, int y) => Main.player[Player.FindClosest(new Vector2(x, y) * 16f, 16, 16)];
+ 
+-	private static void KillTile_DropItems(int x, int y, Tile tileCache, bool includeLargeObjectDrops = false)
```

Comment:
Please add a comment above the method `// made internal, added includeAllModdedLargeObjectDrops param`